src/components/FieldAdapter.js

```diff
@@ -58,14 +61,15 @@ const adapterComponentProps = {
     error: propTypes.bool,
     valid: propTypes.bool,
     loading: propTypes.bool,
-    errorText: propTypes.string,
+    validationText: propTypes.string,
```

Comment:
I think this should be a function that accepts the `error`. Right now this text is always shown, the color is the help text color if there's no error and red if there is an error. If we change this to a function, the app can control if this should be shown when there's no error or not

Reply:
I can't reproduce the behaviour you are describing, and I am also not sure what sort of behavior you'd like to enable by passing a function......
> Right now this text is always shown, the color is the help text color if there's no error and red if there is an error.

I only see the validationText when there is an error. Perhaps this is a bug that comes up in the more elaborate stories that you have created. If so, let's assume it's correct for now, and tackle the bug later.

Reply:
Check out the `Form_guide` branch (from #127).
I've added a `Validation text` story to the `Component: Input` story.
There's no error whatsoever, but the text is shown.

Reply:
> let's assume it's correct for now, and tackle the bug later.

I guess you don't agree?

Reply:
Yeah, I don't really agree, but I think we should talk about intentions first.
What do you want validationText to be? Just like the `helpText` which turns red if the component is invalid to help the user understand how the value is being validated? Then I'd understand why it's visible, even when there's no error. Until now, I thought it's just a custom error message